### ISSUE TITLE
tweakSwapAlmostBrokenTools swaps in the wrong instances

### ISSUE THREAD

**Issue by Likeny3** [maruohon/tweakeroo#144]
tweakSwapAlmostBrokenTools swaps your tools when you use it in instances where it won't use durability. Such as using your low durability pickaxe to open a door. Very annoying and I don't think it's the intended purpose of the tweak.

**Comment by maruohon** [comment]
Yes it's annoying, but to avoid that I would probably have to hard code a list of items and their action types to know when it's safe to not swap since no durability would be used. And then it would only work properly for vanilla items... Although I guess there are some cases that I can also determine from some method overrides existing to try to automatically guess the actions, without having to hard code everything.

**Comment by Likeny3** [comment]
Would be great if it existed at the very least for vanilla items.
